pkg/maps/policymap/policymap.go

```diff
@@ -261,14 +270,14 @@ func (key *PolicyKey) New() bpf.MapKey { return &PolicyKey{} }
 
 // NewKey returns a PolicyKey representing the specified parameters in network
 // byte-order.
-func NewKey(id uint32, dport uint16, proto uint8, trafficDirection uint8) PolicyKey {
+func NewKey(id uint32, dport, mask uint16, proto uint8, trafficDirection uint8) PolicyKey {
```

Comment:
IMHO we should have created a new `NewKeyWithMask` function to avoid refactoring all of the source code.